corpus/literals.txt

```diff
@@ -151,20 +151,6 @@ def foo(a: Char = 'c') = a + 'd'
       (operator_identifier)
       (character_literal))))
 
-==========================
-Symbol literals
-==========================
-
-val mySymbol = 'c
-
-val myOtherSymbol = 'thing
-
----
-
-(compilation_unit
-  (val_definition (identifier) (symbol_literal))
-  (val_definition (identifier) (symbol_literal)))
-
```

Comment:
I am removing symbol literals. This would now be `(quoted_expression (identifier))`